src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import TodoList from './TodoList'
+
+
+
+const Todo = props => {
+    return (
+        <ul>
+            {props.todoList.map(todo => <li onClick={props.taskComplete} completed={todo.completed} name="hello">{todo.task}</li>)}
+        </ul>
+    );
+}
```

Comment:
Don't forget to add a [`key`](https://reactjs.org/docs/lists-and-keys.html#keys) prop in your `map` method.